build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "8.3.0" apply false
+    id("com.android.application") version "8.3.1" apply false
     id("org.sonarqube") version "4.4.1.3373"
 }
```

Comment:
Shouldn't be committed according to github moral values.

Reply:
Yep, thats something that we should probably have added to gitignore file some time ago but will definetly do later